RetailCoder.VBE/Refactorings/Rename/RenameEventHandler.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Rubberduck.Common;
+using Rubberduck.Parsing.Symbols;
+using Rubberduck.UI;
+using System.Diagnostics;
+using System.Linq;
+
+namespace Rubberduck.Refactorings.Rename
+{
+    public class RenameEventHandler : RenameHandlerBase
+    {
+        public RenameEventHandler(RenameModel model, IMessageBox messageBox)
+            : base(model, messageBox) { }
+
+        override public string ErrorMessage
+        {
+            get
+            {
+                return string.Format(RubberduckUI.RenameDialog_EventRenameError, Model.Target.IdentifierName);
+            }
+        }
+
+        override public void Rename()
+        {
+            Debug.Assert(Model.Target.DeclarationType == DeclarationType.Event, "Resolving User Selection to Declaration Failed");
+
+            var eventDeclaration = Model.Target;
+
+            RenameUsages(eventDeclaration, Model.NewName);
+
+            var handlers = Model.Declarations.FindHandlersForEvent(eventDeclaration).ToList();
```

Comment:
`IEnumerable<Declaration>` extension methods predate the MUCH more efficient `DeclarationFinder`. Let's avoid using these extension methods, and delete them once nothing uses them anymore.